### PR TITLE
fix: Fix fails when using tabs minHeight inside the small card

### LINKED ISSUE #48781: <Card /> 组件在 small 尺寸下的卡片式页签出现多余的 border 边框
### Reproduction link

[Edit on CodeSandbox](https://codesandbox.io/p/sandbox/antd-reproduction-template-forked-trt56g)

### Steps to reproduce

查看 <Card /> 组件在 small 尺寸下已激活的卡片式页签

### What is expected?

已激活的卡片式页签下方与内容区无分割

### What is actually happening?

已激活的卡片式页签下方出现边框细线

| Environment | Info |
| --- | --- |
| antd | 5.17.0 |
| React | 18.3 |
| System | Windows 11 |
| Browser | Edge 124.0.2478.80 |

屏幕截图 2024-05-06 130936

屏幕截图 2024-05-06 131157

### PR DESCRIPTION
**Pull request by wanpan11**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

fix: https://github.com/ant-design/ant-design/issues/48781

### 💡 Background and solution
Card 组件在 small 尺寸下的卡片式页签 head minheight 样式权重不够 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Fix the issue of missing bottom border when Card size="small"      |
| 🇨🇳 Chinese |    修复 Card size="small" 时下边框丢失的问题      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/card/__tests__/index.test.tsx [hunk 1]
```diff
@@ -87,6 +87,7 @@ describe('Card', () => {
       </Card>,
     );
     expect(container.querySelectorAll('.ant-tabs-small').length === 0).toBeFalsy();
+    expect(container.querySelector('.ant-card-head')).toHaveStyle('min-height: 0;');
```
[the remark this answers is not in the document]
Comment (wanpan11) [reply components/card/__tests__/index.test.tsx:90]: 好的 我去掉

File: components/card/style/index.ts [hunk 2]
```diff
@@ -365,7 +365,7 @@ const genCardStyle: GenerateStyle<CardToken> = (token): CSSObject => {
 
     [`${componentCls}-contain-tabs`]: {
       [`> ${componentCls}-head`]: {
-        minHeight: 0,
+        minHeight: '0 !important',
```
Comment (li-jia-nan) [inline components/card/style/index.ts:368]: 可以通过修改选择器的方式增加权重吗？

Comment (wanpan11) [reply components/card/style/index.ts:368]: 俺试试

File: components/card/style/index.ts [hunk 3]
```diff
@@ -365,7 +365,7 @@ const genCardStyle: GenerateStyle<CardToken> = (token): CSSObject => {
 
     [`${componentCls}-contain-tabs`]: {
       [`> ${componentCls}-head`]: {
```
Comment (li-jia-nan) [inline components/card/style/index.ts:367]: 这样是不是就可以了：

```suggestion
      [`& > div${componentCls}-head`]: {
```

Comment (wanpan11) [reply components/card/style/index.ts:367]: 是的 👍

`[`> div${componentCls}-head`]`  不需要&符好像也可以

**Review by afc163 — APPROVED** [review]

**Comment by afc163** [comment]
> 修复 card 组件 small 模式下使用 tabs 组件 heard minHeight 样式权重不够问题

这个不是好的 changelog。

可以改成`修复 Card `size="small"` 时下边框丢失的问题。`。

**Pull request merged**